Triphouse-project/index.html

```diff
@@ -47,21 +49,48 @@ <h1>
                       <input type="date" id="out" />
                     </div>
                   </div>
-                  <button>Submit</button>
+                  <button>Search</button>
                 </form>
                 <div class="header__download">
                     <a href="#">
                         <img src="./img/google-play-badge.svg" alt="Google Play"/>
                     </a>
                     <a href="#">
-                        <img src="./img/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.svg" alt="App Store"/>
+                        <img src="./img/app_store.svg" alt="App Store"/>
                     </a>
-                </div>
+                  </div>
               </div>
+           </div>
+        </div>
+        <div class="header__bg_mob">
+            <div class="header__wrapper_mob">
+                <img src="./ikons/logo_mob.svg" alt="">
+
             </div>
-          </div>
+            <div>
+                <h1 class="header_txt_mob">
```

Comment:
![2021-04-22_21h48_39](https://user-images.githubusercontent.com/19935417/115770925-fa60fb80-a3b5-11eb-916c-59faa4dbce17.png)
Why does that visible ?